_lib/wordpress_view_processor.py

```diff
@@ -51,8 +51,9 @@ def process_view(post):
         hero_url = os.path.expandvars("$WORDPRESS/hero/" + hero_id + "/?json=1")
         response = requests.get(hero_url)
         hero_data = json.loads(response.content)
-        hero_data = hero_data['post']
-        hero_data['related_posts'] = hero_data['custom_fields']['related_post']
-        post['hero'] = hero_data
+        if hero_data['status'] is "ok":
```

Comment:
After running `json.loads(json doc)`, you're just dealing with a python dict (which generally prints out with single quotes).  So I don't think we should use double quotes just because it _came from_ JSON.  If we're going to standardize, my vote goes to single quotes because it's 50% of the effort of typing a double. :neckbeard: